lib/vmdb/loggers.rb

```diff
@@ -33,7 +33,17 @@ def self.create_logger(log_file, logger_class = ManageIQ::Loggers::Base)
       log_file = ManageIQ.root.join("log", log_file) if log_file.try(:dirname).to_s == "."
       progname = log_file.try(:basename, ".*").to_s
 
-      logger_class.new(log_file, progname: progname).tap do |logger|
+      manageiq_uid = Process::UID.from_name("manageiq")
+      manageiq_gid = Process::GID.from_name("manageiq")
```

Comment:
TODO memoize these and handle if missing (e.g. dev/test/containers)

Reply:
This should only be on the appliance right?  Also wondering if this should live in manageiq-loggers itself.